content/de/about/vision/vision.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Vision Statement
+description: Die Vision hinter DigiWF.
+category: 'Vision'
+categoryIcon: 'mdi-lightbulb-on'
+position: 1
+---
+
+<figure>
+<v-img alt="Vision statement: For us, low-code means that developers use standard technologies to provide functionality that can be easily reused." contain max-width="960" 
+src="images/about/vision/digiwf_vision.png" 
+lazy-src="images/about/vision/preview_digiwf_vision.png" ></v-img>
+<figcaption>Die DigiWF Vision</figcaption>
+</figure>
+
+In einem agilen Umfeld wollen wir, dass eine Auswahl an Mustern und Technologien - auch Bausteinen - 
+zur Verfügung gestellt wird, mit deren Hilfe Entwickler in der Lage sind genau die Low Code Funktionalitäten zu 
+entwickeln, die in dem Moment in der Organisation gebraucht werden. Das ist das Ziel von DigiWF. Wir 
+stellen nicht eine riesige Auswahl an Features zur Verfügung (und lassen sie uns bezahlen), von denen nur 10% 
+genutzt werden, sondern befähigen eine Entwicklungsorganisation sich selbst eine maßgeschneiderte Lösung (die 
+wiederverwendbar sind) rund um 
+BPMN und DMN zu bauen.
```

Comment:
Würde den zweiten Teil etwas anders formulieren:

In einem agilen Umfeld wollen wir, dass eine Auswahl an Mustern und Technologien - auch Bausteinen - zur Verfügung gestellt wird, mit deren Hilfe Entwickler in der Lage sind, genau die Low Code Funktionalitäten zu entwickeln, die in dem Moment in der Organisation gebraucht werden. Das ist das Ziel von DigiWF. Wir stellen keine proprietäre out-of-box Lösung zur Verfügung, die durch Customizing angepasst wird, sondern befähigen eine Entwicklungsorganisation sich selbst eine maßgeschneiderte Lösung zu bauen. Dabei setzen wird auf wiederverwendbare Low-Code Funktionalitäten, die wir über BPMN und DMN nutzbar machen. So kann mit der Plattform klein gestartet und bei Bedarf schnell skaliert werden.

Reply:
müsste man dann noch ins englische übersetzen 😄

Reply:
Habe ich so eingefügt und für dich übersetzt :smile: